test/unit/TokenTest.php

```diff
@@ -578,6 +578,25 @@ public function isPermittedForShouldReturnTrueWhenAudienceMatches()
         self::assertTrue($token->isPermittedFor('testing'));
     }
 
+    /**
+     * @test
+     *
+     * @covers ::getClaim
+     *
+     * @uses \Lcobucci\JWT\Token::__construct
+     * @uses \Lcobucci\JWT\Token::convertToDataSet
+     */
+    public function isPermittedForWillTriggerDeprecationWhenAudienceIsArray()
+    {
+        $this->expectDeprecation('Behaviour for audience claims has changed in 3.4, you will only get the first array entry as a string. In 4.0 this will change again with more flexibility, please consult the documentation.');
+        $token = new Token(
+            [],
+            [RegisteredClaims::AUDIENCE => ['testing1', 'testing2']]
+        );
+
+        self::assertSame('testing1', $token->getClaim('aud'));
```

Comment:
Same here:
```suggestion
        $token = new Token(
            [],
            [RegisteredClaims::AUDIENCE => ['testing1', 'testing2']]
        );
        
        $this->expectDeprecation('Behaviour for audience claims has changed in 3.4, you will only get the first array entry as a string. In 4.0 this will change again with more flexibility, please consult the documentation.');

        $token->getClaim('aud');
```

Reply:
See above